.gitignore

```diff
@@ -90,6 +90,7 @@ packaging/release/ansible_release
 /test/integration/inventory.remote
 /test/integration/inventory.networking
 /test/integration/inventory.winrm
+/test/integration/inventory.aws_ssm
```

Comment:
Hi @mattclay , 

I think we will need this file, as AWS SSM connection plugin requires the custom inventory file, which has informatiom about instance id, region and s3 bucket. 

Can you please advise, if we remove this from here, what will be better way of adding that?

Reply:
Tests for AWS should include the line `cloud/aws` in the `aliases` file for that test target. That will provide the necessary credentials for connecting to AWS. I recommend taking a look at the [inventory_aws_ec2](https://github.com/ansible/ansible/tree/devel/test/integration/targets/inventory_aws_ec2/) integration test to see how this works.

Reply:
Thanks for reply,

I think I am confused here. Although this is AWS SSM connection plugin, it is pretty much alternative SSH.

Also, this will use custom inventory file, which has instance id. So we need to create those instances before executing tests cases.  As each and every time instance id will change, using template for inventory `/test/integration/inventory.aws_ssm.template` is good approch instead of adding instance id into `/test/integration/targets/connection_aws_ssm/inventory` file

Let me know your thoughts on this.

Reply:
The test should create the instance it needs, run the AWS SSM connection tests, and then remove the instance. You can use a template for the inventory file as part of this process, but any files need by the test should reside under `test/integration/targets/connection_aws_ssm/`.

Reply:
Got it @mattclay, thanks for clarificaiton.  I do have last question before updating files. 

Lets say, I want to execute ping module integration test using AWS SSM Connection plugin. Right now it allows me using command:

```
test/runner/ansible-test aws-ssm-integration -v ping
```

How we can achieve this similar without doing changes in `test/runner/lib/` code?

Reply:
The standard connection plugin integration tests should cover everything needed for basic testing of the connection plugin, so running other integration tests using the connection plugin shouldn't be needed.

That being said, there are some upcoming changes to ansible-test which will allow running integration tests using connection plugins other than `local`, at which point you should be able to run arbitrary tests like `ping` using AWS SSM.

Reply:
Got it. Thanks for reply @mattclay . I will revert all the changes mentioned.

Reply:
@mattclay Reverted all mentioned changes.